lib/utils.js

```diff
@@ -7,98 +10,79 @@ var assert = require("assert");
  * @returns Configuration Object
  */
 exports.parseMongoConnectionURL = function (mongoConnectionURL) {
-  var result = {};
 
-  //The connection string will be of the form mongodb://user:password@host:port,host2port2/databasename?option=someOption
+  //The connection string will be of the form mongodb://username:pass@db1.prv.mbaas2.cluster.feedhenry.net,db2.prv.mbaas2.cluster.feedhenry.net:27017/domain-vvuq7douloe2zkyktub7fcmt-demos-dev?replicaSet=cluster-mbaas2_rs1
```

Comment:
I can revert this, or perhaps use the example from MongoDB docs? `mongodb://[username:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[database][?options]]`

What you think?